ign_ros2_control/src/ign_system.cpp

```diff
@@ -505,6 +631,96 @@ hardware_interface::return_type IgnitionSystem::write(
         *jointEffortCmd = ignition::gazebo::components::JointForceCmd(
           {this->dataPtr->joints_[i].joint_effort_cmd});
       }
+    } else {
+      // Fallback case is a velocity command of zero
+      double target_vel = 0.0;
+      auto vel =
+        this->dataPtr->ecm->Component<ignition::gazebo::components::JointVelocityCmd>(
+        this->dataPtr->joints_[i].sim_joint);
+
+      if (vel == nullptr) {
+        this->dataPtr->ecm->CreateComponent(
+          this->dataPtr->joints_[i].sim_joint,
+          ignition::gazebo::components::JointVelocityCmd({target_vel}));
+      } else if (!vel->Data().empty()) {
+        vel->Data()[0] = target_vel;
+      } else if (!vel->Data().empty()) {
+        vel->Data()[0] = target_vel;
+      }
```

Comment:
I think we can remove this block

```suggestion
      } else if (!vel->Data().empty()) {
        vel->Data()[0] = target_vel;
      }
```

Reply:
[2568e8c](https://github.com/ros-controls/gz_ros2_control/pull/121/commits/2568e8cf443f0e29eecb707df23703599e1621fa)